first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -0,0 +1,630 @@
+    "\n",
+    "publish_version_widget = PublishVersionWidget()\n",
+    "display(publish_version_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add resources to a data collection version\n",
+    "<a id=\"add-resources\"></a>\n",
+    "\n",
+    "Resources in data collections live in \"versions\". To make a resource available in a published data collection version, it must be within the top-level workspace folder corresponding to that version. Controlled and/or referenced resources can be added to the top-level version folder in the workspace corresponding to the data collection created in the previous step in the following ways.\n",
+    "* Navigate to the Workbench Resources tab for the workspace corresponding to your data collection. Move desired resources to the top-level folder named for your version. \n",
+    "* Run the following Workbench CLI commands to get the version folder's ID, then move a resource to the version folder:\n",
+    "```\n",
+    "# Get ID for version folder\n",
+    "terra folder tree\n",
+    "# Move desired resource to version folder\n",
+    "terra resource move --folder-id=<FOLDER_ID> --name=<RESOURCE_NAME> --workspace=<WORKSPACE_ID>\n",
+    "```\n",
+    "* Use the widgets provided in [../workspace_resource_examples.ipynb](../workspace_resource_examples.ipynb../workspace_resource_examples.ipynb) to create controlled and add referenced resources to this workspace."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "\n",
+    "\n",
+    "### Reset your current workspace ID\n",
+    "\n",
+    "When you create a new workspace via the Workbench CLI, your cloud environment will automatically point to that newly-created workspace as your current workspace. This behavior means you can jump right into adding resources once you've created a new workspace without having to point the cloud environment to the new workspace explicitly. However, since you will next convert your newly created workspace into a data collection, you should ensure that you point your cloud environment back to your original, non-data-collection workspace before proceeding further in this notebook.\n",
+    "\n",
+    "Run the cell below to point your cloud environment to the original workspace as your current workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!terra workspace set --id={CURRENT_WORKSPACE_ID}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Add data collection to workspace\n",
+    "<a id=\"add-dc-to-ws\"></a>\n",
+    "\n",
+    "Follow the steps below to add your new data collection to an Workbench workspace.<br>The video below provides a visual walkthrough of these steps.\n",
```

Comment:
It feels like at the end of this section, we should direct them to the UI where they can see that if they click on 'data catalog' they can see what their new DC & version will look like to users.

Reply:
I believe the embedded screencast does just that.

Reply:
IMO it would be helpful to add another line (that links to the UI) for those who prefer text to video-watching-- but your call.